### ISSUE TITLE
CodePeak 2025 : Barcode and QR Code Recognition

### ISSUE THREAD

**Issue by prathimacode-hub** [prathimacode-hub/ML-ProjectKart#618]
Define You:

- [x] CodePeak 2025 Participant 
- [ ] Contributor

Hey OS participants, good to see you.

This project will help us in Recognition of Information stored through Barcode and QR Code, with the help of Python.

Barcode Dataset : https://www.kaggle.com/whoosis/barcode-detection-annotated-dataset
QRcode Dataset : https://www.kaggle.com/coledie/qr-codes

If you like to work on this issue, follow the given guidelines for code as well as for the README. You can try using different algorithms and Python libraries for better modelling practices

**Comment by adi271001** [comment]
@prathimacode-hub please assign this issue to me

**Comment by prathimacode-hub** [comment]
Are you a KWOC participant? @adi271001

**Comment by adi271001** [comment]
@prathimacode-hub yes

**Comment by adi271001** [comment]
actually lats date was yesterday as i confirmed now , is this repo part of swoc or dwoc or jwoc

**Comment by prathimacode-hub** [comment]
It's going to be a part of dwoc @adi271001

**Comment by adi271001** [comment]
please assign this to me under DWOC

**Comment by prathimacode-hub** [comment]
Issue assigned to you. Go ahead. @adi271001

**Comment by Mikeyzgoat** [comment]
hi @prathimacode-hub , I'd like to contribute to this

**Comment by prathimacode-hub** [comment]
Issue assigned to @Mikeyzgoat

**Comment by Mikeyzgoat** [comment]
Hi @prathimacode-hub just wanted to confirm, is the solution expected to have classification of image type into bar code and qr code, or we need to tag bar code and qr codes in a random image. 

Also specify suggested approach:
1. Use YOLO to tag bar code and qr codes
2. Create a CNN, similar models are compare which classifies best.

**Comment by prathimacode-hub** [comment]
Classification of Barcode and QR code and tag in a random image. Choice of models is upto you but use deep learning models minimum of 3-4 algorithms, show comparison along with accuracies and add a summarized statement of what's the best fit model for this solution... @Mikeyzgoat

**Comment by prathimacode-hub** [comment]
Have you created your PR? @Mikeyzgoat

**Comment by Mikeyzgoat** [comment]
yes @prathimacode-hub , i have tagged you in the comments part of the PR

**Comment by prathimacode-hub** [comment]
This issue isn't closed yet, I am reviewing it. But yeah, need additional details in README file and proper ending to the modelling process in your ipynb file. @Mikeyzgoat

**Comment by Mikeyzgoat** [comment]
Ok, will fix it by EOD.

**Comment by prathimacode-hub** [comment]
Sure, any queries feel free to ask @Mikeyzgoat

**Comment by Mikeyzgoat** [comment]
Sure. Check discord pls

**Comment by prathimacode-hub** [comment]
I didn't get any notification @Mikeyzgoat
